File: problem_solving_for_beginners/array_problems/c-programming/sum_of_array.c
#include<stdio.h>

/**
 * main - Sum of the elements of array
 *
 * Return: 0 Always (success)
 */

int main(void)
{
    int arr[10], n, i, sum = 0;
    printf("Enter the size of array: ");
    scanf("%d", &n);

    for(i = 0; i < n ; i++)
    {
        printf("Enter the number of element: ");
        scanf("%d", &arr[i]);
    }

    //Write your code here
    for (i = 0; i < n; i++)
        sum += arr[i];

    printf("Sum = %d\n", sum);
    return 0;
}

File: problem_solving_for_beginners/array_problems/python/sum_of_array.py
# Sum of the elements of array

arr = []
n = int(input("Enter the size of array: "))
sum = 0 

for i in range(n):
    i = int(input("Enter element of array: "))
    arr.append(i)

for i in arr:
    sum += i;

print(f"Sum = {sum:d}")